recipes/pymedphys_analysis/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "pymedphys_analysis" %}
+{% set version = "0.8.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "fa679acffca37cf779dd43466dfdfdd10b8501b874921a91c1a35fb841949098" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
```

Don't make this indirection, directly specify
```suggestion
  sha256: fa679acffca37cf779dd43466dfdfdd10b8501b874921a91c1a35fb841949098
```

---

kk, will do.